RNTester/js/examples/PlatformColor/PlatformColorExample.js

```diff
@@ -261,17 +264,21 @@ function VariantColorsExample() {
     <View style={styles.column}>
       <View style={styles.row}>
         <Text style={styles.labelCell}>
-          {Platform.OS === 'ios'
-            ? "DynamicColorIOS({light: 'red', dark: 'blue'})"
-            : "PlatformColor('?attr/colorAccent')"}
+          {Platform.select({
+            ios: "DynamicColorIOS({light: 'red', dark: 'blue'})",
+            android: "PlatformColor('?attr/colorAccent')",
+            default: 'Unexpected Platform.OS: ' + Platform.OS,
+          })}
         </Text>
         <View
           style={{
             ...styles.colorCell,
             backgroundColor:
               Platform.OS === 'ios'
                 ? DynamicColorIOS({light: 'red', dark: 'blue'})
-                : PlatformColor('?attr/colorAccent'),
+                : Platform.OS === 'android'
```

Comment:
Why not a `Platform.select` here too?

Reply:
In cases where the Platform.select babel transform isn't running, it will attempt to run an undefined function.